EpikCord/flags.py

```diff
@@ -0,0 +1,131 @@
+from __future__ import annotations
+
+from typing import Any, Dict, List
+
+
+class Flag:
```

Comment:
i dont remember why you dont use the `enum.Flag` class?

Reply:
`enum.Flag` has the following syntax:
```py
intents = Intents.A | Intents.B
```
The current implementation would require:
intents = Intents()
intents.a = True
```
I like the latter. I also think that this is good because I want to check if some intents are True (for Voice).